frontend/javascripts/oxalis/view/action-bar/merge_modal_view.tsx

```diff
@@ -141,40 +101,51 @@ class _MergeModalView extends PureComponent<Props, MergeModalViewState> {
     });
   };
 
-  handleMergeProject = async (event: React.SyntheticEvent, isLocalMerge: boolean) => {
+  handleMergeProject = async (event: React.SyntheticEvent) => {
     event.preventDefault();
     const { selectedProject } = this.state;
 
     if (selectedProject != null) {
-      if (isLocalMerge) {
-        const annotation = await getAnnotationCompoundInformation(
-          selectedProject,
-          APIAnnotationTypeEnum.CompoundProject,
-        );
-        this.mergeAnnotationIntoActiveTracing(annotation);
-      } else {
-        const url =
-          `/api/annotations/CompoundProject/${selectedProject}/merge/` +
-          `${this.props.annotationType}/${this.props.annotationId}`;
-        this.merge(url);
-      }
+      const annotation = await getAnnotationCompoundInformation(
+        selectedProject,
+        APIAnnotationTypeEnum.CompoundProject,
+      );
+      this.mergeAnnotationIntoActiveTracing(annotation);
+    }
+  };
+
+  handleImportProject = async (event: React.SyntheticEvent) => {
```

Comment:
looks pretty identical to `handleMergeProject`? did you delete the wrong branch in the if-clause?

Reply:
whoops, so I did! Good catch